rdr_service/data_gen/generators/questionnaires.py

```diff
@@ -691,6 +691,9 @@ def _answer_question(self, question, answer, quantity=None):
     :param quantity: models.quantity.Quantity object
     :return: QuestionnaireResponseGroupQuestion object
     """
+        if question is None:
```

Comment:
I was looking at how the question parameter is generated via _find_question(), e.g:
self._answer_question(self._find_question("ExtraConsent_WelcomeVideo"),...)

Would it be helpful if _find_question() did debug logging when it falls through to its "return None" case on line 871, to log the question_code string that this "code not found" message refers to?

Reply:
Good point, just added the log.